tests/Schema/ComparatorTest.php

```diff
@@ -14,13 +14,22 @@
 use Doctrine\DBAL\Schema\Table;
 use Doctrine\DBAL\Schema\TableDiff;
 use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Types\Types;
 use PHPUnit\Framework\TestCase;
 
 use function array_keys;
 use function get_class;
 
 class ComparatorTest extends TestCase
 {
+    /** @var Comparator */
+    protected $comparator;
+
+    protected function setUp(): void
+    {
+        $this->comparator = new Comparator();
```

Comment:
You have deprecated not passing a platform to the constructor.

Reply:
Yes, but it still has to be covered with a test unless the deprecated code is removed.